custom/icds_reports/templates/icds_reports/mobile_login.html

```diff
@@ -1,12 +1,73 @@
-{% extends "icds/login.html" %}
 {% load i18n %}
-{% block title %}{% trans "Login to ICDS CAS Mobile Dashboard" %}{% endblock title %}
-{% block page_content %}
-{{ block.super }}
+{% load staticfiles %}
+{% load hq_shared_tags %}
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>{% trans "Login to ICDS CAS Mobile Dashboard" %}</title>
+  <link type="text/css" rel="stylesheet" media="all" href="{% static 'bootstrap/dist/css/bootstrap.css' %}"/>
+  <link type="text/css" rel="stylesheet" media="all" href="{% static 'icds_reports/mobile/css/app.css' %}"/>
+  <script src="{% static 'bootstrap/dist/js/bootstrap.js' %}"></script>
+</head>
+<body>
+{% initial_page_data 'implement_password_obfuscation' implement_password_obfuscation %}
```

Comment:
This isn't going to work because the initial page data infrastructure isn't included. Javascript translations also aren't going to work because `statici18n` isn't included. Take a look at the [SMS chat page](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/sms/templates/sms/chat.html) as an example of a page that doesn't inherit from `hqwebapp/base.html`: I think you'll want to add `get_current_language`, the `LANGUAGE_CODE` references, `statici18n`, `javascript_libraries`, `initial_page_data.js`, the `initial-page-data` div and likely the `commcarehq-urls` div.

Reply:
Thanks, super helpful. I've updated this to add a minimal base template and used that in a few places.